addons/binding/org.openhab.binding.paradoxalarm/src/main/java/org/openhab/binding/paradoxalarm/internal/communication/Main.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.paradoxalarm.internal.communication;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.openhab.binding.paradoxalarm.internal.model.ParadoxPanel;
+import org.openhab.binding.paradoxalarm.internal.model.Partition;
+import org.openhab.binding.paradoxalarm.internal.model.RawStructuredDataCache;
+import org.openhab.binding.paradoxalarm.internal.model.Zone;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link Main} - used for testing purposes only.
+ *
+ * @author Konstantin_Polihronov - Initial contribution
+ */
+public class Main {
+
+    private static Logger logger = LoggerFactory.getLogger(Main.class);
+
+    private static final String IP_ADDRESS = "192.168.254.231";
```

Comment:
Is this a default of the device, or something in your own network?

Reply:
Main class is for testing only but it's worth to extract IP address as one of the vararg parameters of the main class. Will be fixed. Thanks.

Reply:
This comment reminded me I needed to do some more reviewing. You should remove this Main class in total. If you want to have some test, it's better to create a real unit test. But you need to create a separate project for it, so that' a little bit more work.

Reply:
Unless it's strictly forbidden to have main class I would prefer to keep it because it eases the debugging of core low-level logic and tranlation into the model without the need to power up openhab and debug using remote aplication with the full stack.